src/main/java/org/takes/facets/hamcrest/HmRqHeader.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.facets.hamcrest;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.takes.Request;
+import org.takes.rq.RqHeaders;
+
+/**
+ * Response Header Matcher.
+ *
+ * <p>This "matcher" tests given response header.
+ * <p>The header name(s) should be provided in lowercase.
+ * <p>The class is immutable and thread-safe.
+ *
+ * @author Eugene Kondrashev (eugene.kondrashev@gmail.com)
+ * @version $Id$
+ * @since 0.23.3
+ * @todo #260:30min Implement additional constructors.
+ * According to #260 there should be also available such constructors:
```

Comment:
@ekondrashev We should indent this line, and the rest of the puzzle (until line 52) one whitespace relative to `@todo` above, like this:

```
@todo #260...
 According to...
```

Otherwise PDD will reject it.